### PR TITLE
Release adaptive

### LINKED ISSUE #2: Add responsiveness on mobile

### PR DESCRIPTION
**Pull request by Kirill-frontend**

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -210,420 +271,788 @@
...
-    display: flex;
-    justify-content: center; }
+  max-width: 100% !important;
+}
+
+.applied .card-wrapper .card {
+  display: -ms-grid;
+  display: grid;
+  grid-auto-flow: column;
+  -ms-grid-columns: (1fr)[4];
```
Comment (harshitkathuria) [inline public/css/main.css]: Please refrain from using [4], instead replace it with repeat().

**Review by harshitkathuria — CHANGES_REQUESTED** [review]
image

The signup section and signup form page going out of viewport. Try to check responsiveness on standard devices.

**Please see comments and update accordingly**

**Comment by Kirill-frontend** [comment]
I have not touched the grid's, and I do not quite understand where it goes off the screen

**Comment by harshitkathuria** [comment]
> I have not touched the grid's, and I do not quite understand where it goes off the screen

The signup section on the landing page has two parts. One for recruiter and another one for students. The students section's signup button is overflowing. And the same goes for signup form.

**Comment by harshitkathuria** [comment]
> I have not touched the grid's, and I do not quite understand where it goes off the screen

Sorry, It was my mistake. Just comment out or replace the [4] with repeat(4, 1fr) and then it will be ready to merged. :)

**Comment by harshitkathuria** [comment]
LGTM, Thanks

**Comment by Kirill-frontend** [comment]
Ok I was glad to help

**Pull request merged**